src/server/io_mgr.h

```diff
@@ -22,12 +22,14 @@ class IoMgr {
   // (io_res, )
   using GrowCb = std::function<void(int)>;
 
-  IoMgr();
+  using ReadCb = std::function<void(int)>;
 
   // blocks until all the pending requests are finished.
   void Shutdown();
 
-  std::error_code Open(const std::string& path);
+  std::error_code Open(std::string_view path);
+
+  std::error_code Grow(size_t len);
```

Comment:
you have Grow and GrowAsync. Please add comments regarding the differrence and also double check if you still need both

Reply:
I need both. I will remove all uneccessary functions when we remove the old tiered storage